src/components/Sidebar/Sidebar.jsx

```diff
@@ -60,28 +61,29 @@ export default class Sidebar extends Component {
 
   componentDidMount() {
     setTimeout(
-      this._recalculate.bind(this),
+      this._recalculate,
       250
     );
 
     document.addEventListener(
       'scroll',
-      this._recalculate.bind(this)
+      this._recalculate
     );
   }
 
   componentWillUnmount() {
     document.removeEventListener(
       'scroll',
-      this._recalculate.bind(this)
+      this._recalculate
     );
   }
 
   /**
    * Re-calculate fixed state and position
    *
    */
-  _recalculate() {
+  _recalculate = throttle(() => {
+    console.log('recalc running');
```

Comment:
Please, clean `console` statements.

Reply:
LOL how did that get through 💃

Reply:
thanks mate, very good catch

Reply:
Not through me. :male_detective: :laughing: